app/modules/import-export/utils.php

```diff
@@ -115,4 +115,13 @@ public static function get_import_sessions( $should_run_cleanup = false ) {
 
 		return $import_sessions;
 	}
+
+	public static function update_space_between_widgets_values( $space_between_widgets ) {
+		$size = strval( $space_between_widgets['size'] );
+		$space_between_widgets['column'] = $size;
+		$space_between_widgets['row'] = $size;
+		$space_between_widgets['isLinked'] = true;
+
+		return $space_between_widgets;
+	}
```

Comment:
@davseve You need a safeguard in place if `$space_between_widgets['size']` does not exist.

Reply:
Done